go/engine/resolve_identify2.go

```diff
@@ -6,15 +6,19 @@ package engine
 import (
 	"github.com/keybase/client/go/libkb"
 	keybase1 "github.com/keybase/client/go/protocol"
+	gregor "github.com/keybase/gregor"
 )
 
 type ResolveThenIdentify2 struct {
 	libkb.Contextified
-	arg      *keybase1.Identify2Arg
-	i2eng    *Identify2WithUID
-	testArgs *Identify2WithUIDTestArgs
+	arg                   *keybase1.Identify2Arg
+	i2eng                 *Identify2WithUID
+	testArgs              *Identify2WithUIDTestArgs
+	responsibleGregorItem gregor.Item
```

Comment:
The `responsibleGregorItem` plumbing duplicates what's in `Identify2WithUID`, which is a little unfortunate. It would be nice to stick that on `Identify2Arg` instead, but that's generated in the protocol, and I don't think we want the keybase1 protocol to depend on the gregor1 protocol.